packages/atlas/src/api/hooks/nfts.ts

```diff
@@ -7,11 +10,73 @@ import {
 } from '@/api/queries'
 import { createLookup } from '@/utils/data'
 
-export const useNft = (id: string) => {
+export type NftStatus =
+  | {
+      status: 'auction'
```

Comment:
Can we also add here type of auction - i.e. `open-auction` and `english-auction`?